### PR TITLE
{bio}[intel/2018a] Canu v1.7

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3169.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/b232b22a064290257b54e8fe7b1a6eb4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2602.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/28d5d5a6510b47dc561c3cc66abf90ad for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login20.cluster.bc2.ch - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/98ae57fb32dbc15d9950bd0170a12558 for a full test report.

File: easybuild/easyconfigs/c/Canu/Canu-1.7-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+easyblock = 'MakeCp'
+
+name = 'Canu'
```
Comment (boegel) [inline easybuild/easyconfigs/c/Canu/Canu-1.7-intel-2018a.eb]: note: renamed to `canu` to match existing easyconfigs in #6389

**Pull request merged**